### ISSUE TITLE
dist.tgz

### ISSUE THREAD

**Issue by jtrujill** [HealthCatalyst/Fabric.Cashmere#215]
I'm not 100% but I imagine this file is generated during the npm publish, correct? This shouldn't be committed to the repo. Every time this file changes it'll add another 1mb to the size of this repo and anything that is generated shouldn't be committed to a repo.

Is there any issue with me adding a gitingore rule for this file? I can't remember if npm uses the .gitignore file for it's publish feature

**Comment by joeskeen** [comment]
Agreed.  Not sure how that got added to the repository, but it should be removed and ignored.
